sirepo/sim_data/flash.py

```diff
@@ -26,8 +28,28 @@ def fixup_old_data(cls, data):
             )
 
     @classmethod
-    def proprietary_lib_file_basename(cls, data):
-        return '{}.zip'.format(sirepo.util.secure_filename(data.models.simulation.flashType))
+    def flash_exe_path(cls, data):
+        from pykern import pkio
+        import distutils.spawn
+        p = distutils.spawn.find_executable(
+            '{}-{}'.format(
```

Comment:
save in `n`

Reply:
https://github.com/radiasoft/sirepo/pull/2490/commits/b2b8aee87b4188997b30427644819f981d3d47e2